### PR TITLE
feature: add exporter for mysql addon

### LINKED ISSUE #919: [Improvement] add mysql-exporter for MYSQL addon

### PR DESCRIPTION
**Pull request by kubeJocker**
fix #919

### CODE REVIEW

File: addons/mysql/templates/backuppolicytemplate.yaml [hunk 1]
```diff
@@ -41,43 +40,12 @@ spec:
...
+    schedules:
     - backupMethod: xtrabackup
       enabled: false
       cronExpression: "0 18 * * 0"
       retentionPeriod: 7d
     - backupMethod: volume-snapshot
       enabled: false
       cronExpression: "0 18 * * 0"
-      retentionPeriod: 7d
-  - componentDefRef: mysql
```
Comment (nayutah) [inline addons/mysql/templates/backuppolicytemplate.yaml:53]: why delete this part ?

Comment (nayutah) [reply addons/mysql/templates/backuppolicytemplate.yaml:53]: it is obsolete in 0.9

File: addons/mysql/templates/_helpers.tpl [hunk 2]
```diff
@@ -394,3 +399,72 @@ memberLeave:
...
+  - name: HA_COMPNENT
+    value: orchestrator
+{{- end -}}
+
+{{- define "mysql.spec.runtime.exporter" -}}
+command:
+  - bash
+  - -c
+  - |
+    mysqld_exporter --mysqld.username=${MYSQLD_EXPORTER_USER} --web.listen-address=:${EXPORTER_WEB_PORT} --log.level=info
```
Comment (nayutah) [inline addons/mysql/templates/_helpers.tpl:453]: make log level configurable in values.yaml to help debug

**Review by shanshanying — APPROVED** [review]

File: addons/mysql/templates/_helpers.tpl (hunk 2 above)
Comment (kubeJocker) [reply addons/mysql/templates/_helpers.tpl]: done

**Pull request merged**